Mail/smtp.php

```diff
@@ -371,9 +379,12 @@ protected function send_or_fail($recipients, $headers, $body)
 
         /* Send the message's headers and the body as SMTP data. */
         $res = $this->_smtp->data($body, $textHeaders);
-        list(,$args) = $this->_smtp->getResponse();
 
-        if (preg_match("/ queued as (.*)/", $args, $queued)) {
+        list($code,$args) = $this->_smtp->getResponse();
+
+        $this->response = $code . ' ' . $args;
+
+        if (preg_match("/ queue (.*)| queued (.*)/i", $args, $queued)) {
```

Comment:
Is it better to look for `/ queued as (.*)| queue (.*)/`; that way there's no chance of picking up the "as" as the first match?
Behaviour would be unchanged for existing users, and would also support gmail going forward

Reply:
@CloCkWeRX Thank you for your critical PR. After a little bit of playing around with some preg_match examples. I decided to let this passage untouched, because of backwards compatibility. After the `queued as` stands a queue id normaly. in responses of other mailservers including `queued` or `queue` the following informations are not only the queue id. So the users who uses this property at the moment, gets more than one word or id in the future. that is not backwards compatible.